src/app/parcels/ActionBar/Statuses.tsx

```diff
@@ -152,7 +152,7 @@ const Statuses: React.FC<Props> = ({
             setServerErrorMessage(`${getStatusErrorMessage(error)} Log ID: ${error.logId}`);
         }
         hasAttemptedToSaveParcelStatus();
-        setStatusModal(false);
+        error ?? setStatusModal(false);
```

Comment:
nitpick: maybe this is just a personal preference, but IMO `??` and `&&` etc form expressions, not a conditional execution of tasks. To me it reads better to use `if` statements, even if at the cost of extra lines.

```suggestion
        if (!error) {
            setStatusModal(false);
        }
```

Reply:
have done